Atomic/syscontainers.py

```diff
@@ -479,6 +488,10 @@ def _checkout(self, repo, name, img, deployment, upgrade, values=None, destinati
             raise e
 
     @staticmethod
+    def _template_has_stoppost_and_pidfile(template):
+        return "$EXEC_STOPPOST" in template and "PIDFILE" in template
```

Comment:
at the moment it is not used, I've added it only for completeness in case we will need it in future.

I've based the template on the example at the bottom of http://runc.io/